app/components/wild-card.js

```diff
@@ -1,13 +1,13 @@
 import Ember from 'ember';
```

Comment:
Isn't the wild-card chart used to show the loading circle before data is loaded?